### PR TITLE
Add relnotes for 1.15.0-beta.1

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

File: docs/releases/1.15-NOTES.md [hunk 1]
```diff
@@ -277,3 +277,43 @@
 * fix static check error in vfssync.go [@beautytiger](https://github.com/beautytiger) [#7482](https://github.com/kubernetes/kops/pull/7482)
 * fix(upup/models/cloudup/resources/addons/coredns.addons.k8s.io) missing resourceVersion [@phspagiari](https://github.com/phspagiari) [#7477](https://github.com/kubernetes/kops/pull/7477)
 * modify-doc-small-mistake [@tanjunchen](https://github.com/tanjunchen) [#7519](https://github.com/kubernetes/kops/pull/7519)
+BREAK
```
Comment (rifelpet) [inline docs/releases/1.15-NOTES.md]: Should there be a new heading here instead of `BREAK`?

**Comment by rifelpet** [comment]
Also @justinsb is it worth adding a note about downgrading from 1.15 to 1.14? `kops1.14 update cluster` wont work after having done `kops1.15 update cluster` due to the apiGroup renaming so users should backup their manifest files and use `kops1.14 replace -f` if they need to downgrade

**Comment by justinsb** [comment]
Thanks @rifelpet - I fixed the section header and added a "breaking change" section just to make it explicit at the top.

I also pasted these notes into the release page: https://github.com/kubernetes/kops/releases/tag/1.15.0-beta.1

**Pull request merged**